docs/user_docs/guides/working-with-k8s/6-set-up-operational-rules.md

```diff
@@ -0,0 +1,97 @@
+# Set up operational rules
+
+You can set up operational rules in the `AppConfiguration` model via the `opsRule` field and corresponding platform configurations in the workspace directory. The `opsRule` is the collection of operational rule requirements for the application that are used as a preemptive measure to police and stop any unwanted changes.
+
+## Prerequisites
+
+Please refer to the [prerequisites](1-deploy-application.md#prerequisites) in the guide for deploying an application.
+
+The example below also requires you to have [initialized the project](1-deploy-application.md#initializing) using the `kusion init` command, which will generate a [`kcl.mod` file](1-deploy-application.md#kclmod) under the project directory.
+
+## Example
+
+### App Configuration
+
+ Add the `opsRule` snippet to the `AppConfiguration` in `dev`/main.k`
+
+```py
+import catalog.models.schema.v1 as ac
+import catalog.models.schema.v1.workload as wl
+import catalog.models.schema.v1.workload.container as c
+import catalog.models.schema.v1.trait as t
+
+helloworld: ac.AppConfiguration {
+    workload: wl.Service {
+        containers: {
+            "helloworld": c.Container {
+                image: "gcr.io/google-samples/gb-frontend:v4"
+                resources: {
+                    "cpu": "500m"
+                    "memory": "512M"
+                }
+            }
+        }
+        replicas: 2
+    }
+    # config the maxUnavailable rule
+    opsRule = t.OpsRule {
+        maxUnavailable: "30%"
+    }
+}
+```
+
+### Platform configuration
+
+For platform engineers, you can also set default values of the opsRule in the `~/.kusion/workspace/dev.yaml` to standardize the behavior of applications. The `maxUnavailable` in the platform configuration will be overridden by the value set in the application configuration.
+
+:::info
+If the platform engineers have set the default workload to [Kusion Operation](https://github.com/KusionStack/operating) and installed the Kusion Operation controllers properly, the `opsRules` module will generate a [PodTransitionRule](https://www.kusionstack.io/docs/operating/manuals/podtransitionrule) instead of updating the `maxUnavailable` value in the deployment
+:::
+
+```yaml
+modules:
+    opsRule:
+        default:
+            maxUnavailable: "40%"
+```
+
+## Applying
+
+Re-run steps in [Applying](/docs/user_docs/guides/working-with-k8s/deploy-application#applying), resource scaling is completed.
```

Comment:
Can be simplified as `[Applying](deploy-application.md#applying)`, more see the best practice: [versioning#link-docs-by-file-paths](https://docusaurus.io/docs/versioning#link-docs-by-file-paths)